packages/qri-node/__tests__/qri-node.test.js

```diff
@@ -1,7 +1,64 @@
-'use strict';
+'use strict'
 
-const qriNode = require('..');
+const fs = require('fs-extra')
+const { customAlphabet } = require('nanoid')
+const ObjectsToCsv = require('objects-to-csv')
 
-describe('@qri-io/qri-node', () => {
-    it('needs tests');
-});
+const qri = require('..')
+
+// only use lowercase letters
+const nanoid = customAlphabet('abcdefghijklmnopqrstuvwxyz', 10)
+
+const csvPath = `tmp/body.csv`
+
+describe('@qri-io/qri-node',() => {
+  beforeAll(async () => {
+    // make a tmp dir, store a simple csv in it
+    fs.ensureDir('tmp')
```

Comment:
This is creating the tmp directory to store the csv file, however it's also creating datasets and references in the actual repository of the user running the test. This means the tests will always fail for someone who hasn't run `qri setup`, and those who have will get random dataset names added. A better approach would be to create a "qri" folder inside "tmp", set QRI_PATH to use that folder, and run `qri setup` (with a random username) before running the tests.

Reply:
Great idea.  It's true, I have like 140 randomly named datasets in a real repo because of this.